scenarios/install_terraform.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
maybe better keep it in root?

Reply:
Maybe move it to `scenarios/base` together with `get_project_root`?

Reply:
Updated scripts structure